iroh/src/sync_engine/live.rs

```
@@ -506,8 +528,16 @@ impl<S: store::Store, B: baomap::Store> Actor<S, B> {
         Ok(())
     }
 
+    fn get_replica_if_syncing(&self, namespace: &NamespaceId) -> Option<Replica<S::Instance>> {
+        if !self.syncing_replicas.contains(namespace) {
+            None
+        } else {
+            self.replica_store.open_replica(namespace).ok().flatten()
```

should probably return the error or at least log it